### PR TITLE
✨ add champion repository #22

### LINKED ISSUE #22: Create champion repository

### PR DESCRIPTION
**Pull request by iurynogueira**
fix #22

### CODE REVIEW

**Review by mizek1 — CHANGES_REQUESTED** [review]

**Comment by mizek1** [comment]
There are some files that end line in the end of file is missing.
Prettier didn't run?

**Comment by iurynogueira** [comment]
> Prettier didn't run?

I will check here.

File: src/app/data/repository/champion/champion-repository.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  mapperToObject(champion: ChampionEntity, key: string) {
+    return new ChampionEntity({key, ...champion});
+  }
+
+  getAll(): Observable<ChampionEntity[]> {
+    return this.http
+      .get<{data: {[championName: string]: ChampionEntity}}>(environment.serverUrl + '/champion.json')
+      .pipe(map(({ data }) => Object.keys(data).map((key: string) => {
+        return this.mapperToObject(data[key], key);
```
Comment (lfontesc) [inline src/app/data/repository/champion/champion-repository.service.ts:22]: nice

File: src/app/app.component.ts [hunk 2]
```diff
@@ -9,9 +9,11 @@ import { MangaRepositoryService } from './data/repository/manga/manga-repository
 export class AppComponent implements OnInit {
   title = 'mangas';
```
Comment (lfontesc) [inline src/app/app.component.ts:10]: change

**Comment by iurynogueira** [comment]
> There are some files that end line in the end of file is missing. Prettier didn't run?

fixed!

**Review by lfontesc — APPROVED** [review]

**Review by mizek1 — APPROVED** [review]

**Pull request merged**